sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py

```diff
@@ -146,6 +147,31 @@ def _create_handler(self):
             self.source = self.redirected.address
         await super(EventHubConsumer, self)._open(timeout_time)
 
+    @_retry_decorator
+    async def _receive(self, **kwargs):
```

Comment:
You should take *args as the first parameter, or else positional parameters will not be passed through/the decorator won't work.